tslearn/tests/test_utils.py

```diff
@@ -46,6 +48,7 @@ def test_label_categorizer():
 
 
 def test_conversions():
+    pytest.importorskip('pandas')
```

Comment:
Hum, I have a concern here: I would prefer that we skipped only pandas-related conversion functions (ie. those involving sktime, pyflux and tsfresh) only. Maybe we could split the tests into two functions (one with pandas required, one that should pass even without pandas)?